### PR TITLE
[Dashboard] User approval test plan

### LINKED ISSUE #6577: [User Accounts]Test Plan # 36(widget Registration to Dashboard) Failed
**Describe the bug**
Widget registration on the dashboard page cannot be tested/failed. 
The number of pending account approvals that should be displayed in the My Task panel of the dashboard page is not visible/seen even if Pending Approval set to 'Yes' users are there in user accounts

**Steps to reproduce the behavior (attach screenshots if applicable):**
Go to user accounts and Site set to the user's site and Pending Approval set to 'Yes'.  Check to see if the number of pending account approvals is displayed in the My Task panel of the dashboard page.

### PR DESCRIPTION
**Pull request by laemtl**
In Dashboard > My Tasks 
~~I updated the query executed for displaying the users with Site set to the current user's site and Pending Approval set to 'Yes'. Previously, the query was targeting users from all sites except Data Coordinating Center and Pending Approval set to 'Yes'.~~

I updated the test plan to indicate that pending users can belong to any sites except Data Coordinating Center.

 Resolves #6577

### CODE REVIEW

**Comment by laemtl** [comment]
The change is breaking an integration test:
1) DashboardTest::testPendingUser
Failed asserting that '2' contains "1".
/app/test/integrationtests/LorisIntegrationTest.class.inc:700
/app/modules/dashboard/test/DashboardTest.php:504

**Comment by christinerogers** [comment]
I re-started travis just in case... 
@laemtl do you think there's improvement we could make to the DashboardTest::testPendingUser function? 

It's not actually likely that DCC users are Pending 'Yes'  (DCC is our staff, typically), so not sure this is truly valid test case or a priority, but we should find a way to resolve this smoothly for the release anyway. 

cc @AlexandraLivadas

**Comment by laemtl** [comment]
I think this PR should be closed. The test plan seems out of date and should be edited instead.

**Comment by ridz1208** [comment]
@laemtl adding blocked until we figure out what we are doing with the dashboard testplan.

**Comment by christinerogers** [comment]
@laemtl i think we can resolve/close this issue prettily readily - given what I think you found in the codebase regarding recent changes in the code. 
Can you please link here what you found? i.e. were there any deliberate changes to the behaviour that indicate we should simply update the test plan.  Knowing this would save everyone time on Monday... thanks!

File: modules/user_accounts/test/TestPlan.md [hunk 1]
```diff
@@ -81,9 +81,5 @@ When creating or editing a user: (subtest: edit_user)
 
 ### Widget registration on the dashboard page
 
-36. Verify that if a user has 'User Management / Survey Participant Management' permission, the number of pending
-    account approvals is displayed in the My Task panel of the dashboard page. This should be the number of entries
-    in the User Account page with the following Selection Filter: Site set to the user's site and Pending Approval
-    set to 'Yes'. The Site displayed will be 'All user sites'. Check that you are taken to that page (with the
-    Selection Filter correctly set) when you click on the task.
+36. Verify that if a user has 'Across all sites create and edit users' permission, the number of pending account approvals is displayed in the My Task panel of the dashboard page. This should be the number of entries in the User Account page with the following Selection Filter: Site set to any site except Data Coordinating Center and Pending Approval set to 'Yes'. The Site displayed will be 'All user sites'. Check that you are taken to that page (with the Selection Filter correctly set) when you click on the task.
```
Comment (driusan) [inline modules/user_accounts/test/TestPlan.md:84]: These are 2 different tests. If you have across all sites, there should be no site taken into account, and if you have user accounts without across all sites, it should only be your sites.

**Comment by laemtl** [comment]
@christinerogers Behaviour was updated here: 
- [removed lines](https://github.com/aces/Loris/commit/2ba58824bc4fc99fd6b2dc971ed91360e124151f#diff-8c9451d7355d5c1a8c5afdb8c8556c23R98-R105)
-[added lines](https://github.com/aces/Loris/commit/2ba58824bc4fc99fd6b2dc971ed91360e124151f#diff-82bd08eacfd3bcbc77ecda4d6b83dd0fL177-L198)

File: modules/user_accounts/test/TestPlan.md (hunk 1 above)
Comment (laemtl) [reply modules/user_accounts/test/TestPlan.md]: Is user_accounts_multisite sufficient to access the user_accounts page? Currently, a user with `user_accounts_multisite` and without `user-accounts` permission gets a `You do not have access to this page` on User Accounts page.

File: modules/user_accounts/test/TestPlan.md [hunk 2]
```diff
@@ -81,9 +81,9 @@ When creating or editing a user: (subtest: edit_user)
 
 ### Widget registration on the dashboard page
 
-36. Verify that if a user has 'User Management / Survey Participant Management' permission, the number of pending
-    account approvals is displayed in the My Task panel of the dashboard page. This should be the number of entries
-    in the User Account page with the following Selection Filter: Site set to the user's site and Pending Approval
-    set to 'Yes'. The Site displayed will be 'All user sites'. Check that you are taken to that page (with the
-    Selection Filter correctly set) when you click on the task.
+36. Verify that if a user has `user_accounts_multisite` permission, the number of pending account approvals is displayed in the My Task panel of the dashboard page. This should be the number of entries in the User Account page with the following Selection Filter: Site set to all sites except Data Coordinating Center and Pending Approval set to 'Yes'. The Site displayed will be 'All'. Check that you are taken to that page (with the Selection Filter correctly set) when you click on the task.
```
Comment (christinerogers) [inline modules/user_accounts/test/TestPlan.md:84]:
```suggestion
36. Verify that if a user has `user_accounts_multisite` permission, the correct number of pending account approvals is displayed in the My Task panel of the dashboard page. This should be the number of entries in the User Account page with the following Selection Filter: Site set to all sites except Data Coordinating Center and Pending Approval set to 'Yes'. The Site displayed will be 'All'. Check that you are taken to that page (with the Selection Filter correctly set) when you click on the task.
```

**Pull request merged**